Updated homepage anchor tag href

anilnatha opened the pull request:
Added vanilla JavaScript that alters the homepage anchor HREF value in the header based on the current environment.

If you feel that the following lines of code are redundant, we can remove them.

```javascript
if( authHost === 'auth.maap-project.org') {
    homeHost = 'www.' + homeHost;
}
```

I tested this code using browser developer tools with success since I don't have an AWS account set up to deploy and test this code.

anilnatha committed:
Added Javascript that alters the homepage anchor href value in the header based on the current environment.

bsatoriu reviewed (approved):
Looks great--I'll deploy to DIT for testing.

anilnatha commented:
@bsatoriu I just checked the DIT Auth environment and I see the anchor tag change where I added `id="home-anchor"`, but the JavaScript I added to the `head` tag is missing. And in hindsight, I should have added the code so that it is loaded at the bottom of the page where the other JS code resides, but after inspecting the footer fragment, I don't see any of the code shown in the browser visible in the footer fragment.  I'll look at this again and report back.

bsatoriu commented:
> @bsatoriu I just checked the DIT Auth environment and I see the anchor tag change where I added `id="home-anchor"`, but the JavaScript I added to the `head` tag is missing. And in hindsight, I should have added the code so that it is loaded at the bottom of the page where the other JS code resides, but after inspecting the footer fragment, I don't see any of the code shown in the browser visible in the footer fragment. I'll look at this again and report back.

@anilnatha looks like we discovered the same thing. I moved this code to the body already and deployed to DIT, so we should be good now.

anilnatha commented:
Thanks @bsatoriu!   I am still curious to know how the JS code is being inserted at the bottom of the page, I'll keep digging.  I will delete the issue branch I created.
